src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -857,12 +866,16 @@ void network::configure_primitives_second_output() {
 }
 
 void network::build_insts_deps() {
+    GPU_DEBUG_GET_INSTANCE(debug_config);
+    GPU_DEBUG_CODE(cldnn::instrumentation::mem_usage_logger mem_logger{"build_insts_deps", debug_config->verbose >= 2});
     for (auto& inst : _primitives) {
         inst.second->build_deps();
     }
 }
 
 void network::build_exec_order() {
+    GPU_DEBUG_GET_INSTANCE(debug_config);
+    GPU_DEBUG_CODE(cldnn::instrumentation::mem_usage_logger mem_logger{"build_exec_order", debug_config->verbose >= 2});
```

Comment:
I think it can be wrapped with some helper macro like
`GPU_DEBUG_MEM_LOGGER("name");`
to avoid code duplication and have shorter form

Reply:
Agree, fixed